test/extended/networking/network_segmentation.go

```diff
@@ -361,7 +361,7 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 					),
 				)
 				DescribeTable(
-					"isolates overlapping CIDRs",
+					"isolates overlapping CIDRs [OCPFlaky]",
```

Comment:
-1 to changing test names, can we maintain a list? Similar to the old canary suite Ben had created

Reply:
It's in the minimal.go

https://github.com/openshift/origin/pull/26528/files